### DISCUSSION TITLE
HyperHDR on RPI Boot

### DISCUSSION THREAD

**Discussion by Eiscoffee** [awawa-dev/HyperHDR#216] [Q&A] [answered]
Hello,

I bought a new rpi 4 because my 3b+ stopped working and now i wanted to just use hyperHDR for ambilight on the TV.
So I thought i can pick the hyperhdr Image package to install on the SDcard for the pi, didnt work, strange error, no idea...
(maybe bad SDcard, dont know...)

Then i used Raspberry Pi OS and installed the hyperHDR.deb and it works - until i reboot.
HyperHDR wont start with the Pi OS boot, any ideas what to do?

Thank you!

**Comment by awawa-dev** [comment, marked as answer]
For Rpi 4 any of image can be use:  SD-card-image-rpi012-armv6.zip SD-card-image-rpi234-aarch64 SD-card-image-rpi34-armv7 but aarch64 version is recommended.

Maybe SD card is is dying? Please use [https://www.balena.io/etcher/]( https://www.balena.io/etcher/) to write images in your case. It automatically verifies the SD card content after it's written.

Hard to tell what is happening now with your system. Seems the service didn't started but it's registered and worked after the initial setup. Try to run /usr/bin/hyperhdr manually and check out the possible errors. Also verify dmesg log for system errors.

**Comment by Eiscoffee** [comment]
Okay yes it was the SD card in first case, i just installed the hyperhdr image to a new one and it worked.
ty
